functions/influxdb+grafana.bash

```diff
@@ -291,7 +292,7 @@ grafana_install(){
 
   cond_echo "\\nRestarting Grafana... "
   if ! cond_redirect systemctl restart grafana-server.service; then echo "FAILED (restart service)"; return 1; fi
-  if (curl -4 --retry 6 --retry-connrefused --silent --head http://localhost:3000); then echo "OK"; else echo "FAILED (wait for Grafana to start)"; return 1; fi
+  if (curl -4 -m 120 --retry 20 --retry-connrefused --silent --head http://localhost:3000); then echo "OK"; else echo "FAILED (wait for Grafana to start)"; return 1; fi
```

Comment:
This seems like a lot more time, what changed from before to cause it to no longer work in a timely fashion?

Reply:
frankly I don't know

Reply:
Actually the same code was in there for InfluxDB, too, and the BATS test for InfluxDB failed in our CI.
After I applied the same fix it now works. Fix for both Influx and Grafana are part of the PR.